.travis.yml

```diff
@@ -2,6 +2,8 @@
 # RVM/bundler/ruby and whatnot. Right now 'rust' as a language actually
 # downloads a rust/cargo snapshot, which we don't really want for building rust.
 language: c
+compiler: /usr/bin/gcc-4.7
```

Comment:
Yeah, if we don't set this it will inject `export CC = gcc` *after* our matrix is injected.